core/arch/riscv/plat-spike/conf.mk

```diff
@@ -73,3 +73,4 @@ CFG_TDDRAM_SIZE  ?= 0x00f00000
 CFG_SHMEM_START  ?= 0x7fe00000
 CFG_SHMEM_SIZE   ?= 0x00200000
 CFG_TEE_RAM_VA_SIZE ?= 0x00400000
+CFG_RISCV_M_MODE ?= y
```

Comment:
Something I missed in "riscv: core: riscv.mk: select privilege mode of OP-TEE core"

Can Spike work in S mode?
If it can, this line is not needed because it is the default set in `riscv.mk`
If it cannot then this line should be:
```
$(call force,CFG_RISCV_M_MODE,y)
```
(otherwise someone could do `make M=n S=y` and the checks in `riscv.mk` would not complain, not knowing that this config is not supported by the platform)

Reply:
Yes, Spike supports all privilege levels.